edge/cmd/edgecore/app/server.go

```diff
@@ -0,0 +1,51 @@
+package app
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/kubeedge/beehive/pkg/core"
+	"github.com/kubeedge/kubeedge/edge/pkg/common/dbm"
+	"github.com/kubeedge/kubeedge/edge/pkg/devicetwin"
+	"github.com/kubeedge/kubeedge/edge/pkg/edged"
+	"github.com/kubeedge/kubeedge/edge/pkg/edgehub"
+	"github.com/kubeedge/kubeedge/edge/pkg/eventbus"
+	"github.com/kubeedge/kubeedge/edge/pkg/metamanager"
+	"github.com/kubeedge/kubeedge/edge/pkg/servicebus"
+)
+
+// NewEdgeCoreCommand create edgecore cmd
+func NewEdgeCoreCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "edgecore",
+		Long: `Edgecore is the core edge part of KubeEdge, which contains six modules: devicetwin, edged, 
+edgehub, eventbus, metamanager, and servicebus. Edged is an agent that runs on edge nodes and 
+manages containerized applications and devices. Edgehub is a web socket client responsible for 
+interacting with Cloud Service for the edge computing (like Edge Controller as in the KubeEdge 
+Architecture). This includes syncing cloud-side resource updates to the edge, and reporting 
+edge-side host and device status changes to the cloud. EventBus is a MQTT client to interact with MQTT 
+servers (mosquito), offering publish and subscribe capabilities to other components.  ServiceBus is 
```

Comment:
Duplicate spaces here, and please Corresponding to the module order above, `devicetwin, edged, 
edgehub, eventbus, metamanager, and servicebus.`